src/main/scala/com/campudus/tableaux/database/domain/column.scala

```diff
@@ -709,15 +709,23 @@ sealed trait ConcatenateColumn extends ColumnType[JsonArray] {
 
 case class ConcatColumn(
     override val columnInformation: ConcatColumnInformation,
-    override val columns: Seq[ColumnType[_]]
+    override val columns: Seq[ColumnType[_]],
+    formatPattern: Option[String]
 )(implicit override val roleModel: RoleModel, val user: TableauxUser) extends ConcatenateColumn {
   override val kind: ConcatType.type = ConcatType
 
-  override def getJson: JsonObject =
-    super.getJson.mergeIn(
-      Json.obj("concats" -> columns.map(_.getJson))
-    )
+  override def getJson: JsonObject = {
+    val concatsJson = Json.obj("concats" -> columns.map(_.getJson))
+
+    val formatPatternJson = formatPattern match {
+      case Some(pattern) => Json.obj("formatPattern" -> pattern)
+      case None => Json.emptyObj()
+    }
```

Comment:
In der concat column gibst du das format pattern nur zurück, wenn es nicht null ist. 
Von daher würde ich es in der table (siehe comment in ) auch nicht machen. Damits halt irgendwie gleich ist.

Vom naming her verstehe ich, dass du es hier nicht `concatFormatPattern` nennst, weil das durch den context ja schon klar ist. Für Außenstehende ist das evtl. verwirrend...? Ich finde es so, wie du es gemacht hast, aber fast klarer. Passt also für mich.